APMrover2/mode.h

```diff
@@ -481,3 +485,27 @@ class ModeFollow : public Mode
 
     bool _enter() override;
 };
+
+class ModeSimple : public Mode
+{
+public:
+
+    uint32_t mode_number() const override { return SIMPLE; }
+    const char *name4() const override { return "SMPL"; }
+
+    // methods that affect movement of the vehicle in this mode
+    void update() override;
+    void init_heading();
```

Comment:
init_heading is `public`?

Reply:
yes, init_heading is public so that it can be called at the moment the vehicle is armed.